source/jsonnet/england-wales/census_individual.jsonnet

```diff
@@ -72,7 +72,9 @@ local job_title = import 'individual/blocks/employment/job_title.jsonnet';
 local jobseeker = import 'individual/blocks/employment/jobseeker.jsonnet';
 local main_employment_block = import 'individual/blocks/employment/main_employment_block.jsonnet';
 local main_job_type = import 'individual/blocks/employment/main_job_type.jsonnet';
+local mainly_work_uk = import 'individual/blocks/employment/mainly_work_uk.jsonnet';
 local supervise = import 'individual/blocks/employment/supervise.jsonnet';
+local work_outside_uk = import 'individual/blocks/employment/work_outside_uk.jsonnet';
```

Comment:
Can I be petty and ask that these be named similar?
ie `mainly_work_in_uk`/`mainly_work_outside_uk` etc

Reply:
Not petty - I was about to say the same thing. Consistency helps a lot when going through these jsonnet files